### PR TITLE
Build related fixes

### PR DESCRIPTION
**Pull request by kraj**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -386,7 +386,7 @@ $(STATIC_OUTDIR)/write_batch_test:db/write_batch_test.cc $(STATIC_LIBOBJECTS) $(
 $(STATIC_OUTDIR)/memenv_test:$(STATIC_OUTDIR)/helpers/memenv/memenv_test.o $(STATIC_OUTDIR)/libmemenv.a $(STATIC_OUTDIR)/libleveldb.a $(TESTHARNESS)
 	$(XCRUN) $(CXX) $(LDFLAGS) $(STATIC_OUTDIR)/helpers/memenv/memenv_test.o $(STATIC_OUTDIR)/libmemenv.a $(STATIC_OUTDIR)/libleveldb.a $(TESTHARNESS) -o $@ $(LIBS)
 
-$(SHARED_OUTDIR)/db_bench:$(SHARED_OUTDIR)/db/db_bench.o $(SHARED_LIBS) $(TESTUTIL)
+$(SHARED_OUTDIR)/db_bench:$(SHARED_OUTDIR)/db $(SHARED_OUTDIR)/db/db_bench.o $(SHARED_LIBS) $(TESTUTIL)
```
Comment (shr-project) [inline Makefile]: Is this correct? The missing prerequisite is for db_bench.o target, not $(SHARED_OUTDIR)/db_bench

I think the easiest fix is to include whole SHARED_ALLOBJS which already requires SHARED_OBJDIRS

Comment (shr-project) [reply Makefile]: Or just another explicit rule:

$(SHARED_OUTDIR)/db/db_bench.o: | $(SHARED_OUTDIR)/db

**Comment by cmumford** [comment]
Thanks for the PR @kraj. We switched to CMake as well as some other changes that fix the issues in this PR.

**Comment by kraj** [comment]
@cmumford  I hope the CMake transition helped the logic detecting SSE to be x86 specific as done in first patch.

**Comment by pwnall** [comment]
The code that cares about your SSE support is now in google/crc32c.

**Comment by kraj** [comment]
Thanks @pwnall

**Pull request closed without merging**